sigma/plugins/utility/other/define.py

```diff
@@ -56,12 +56,16 @@
             else:
                 definition += tag.next_sibling
                 definition += " "
-    except:
-        cmd.log('The website has changed; either "intro-colon" or "sx-link sc" are not present.')
+    except IndexError:
+        cmd.log('The website has changed; "sx-link sc" is not present.')
+        return
+    except ValueError:
```

Comment:
The string you specify as a parameter of the exceptions such as ValueError(...) or Exception(...) or any other can be retrieved by casting the exception into a string. So just keep one, change to Exception and print whatever it is.
```
except Exception as e
      cmd.log(e)
      return
```